manifests/resource/server.pp

```diff
@@ -251,6 +256,8 @@
   $client_max_body_size                                                          = undef,
   Optional[Variant[Array[String], String]] $raw_prepend                          = undef,
   Optional[Variant[Array[String], String]] $raw_append                           = undef,
+  Optional[Variant[Array[String], String]] $raw_ssl_prepend                      = undef,
```

Comment:
Can you enforce the minimal string length? I think empty strings don't make sense here:
```suggestion
  Optional[Variant[Array[String[1]], String[1]]] $raw_ssl_prepend                = undef,
```